bio/grit_genomics/wrapper.py

```
@@ -0,0 +1,38 @@
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+command = snakemake.params["command"]
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+input_args = ""
+output_args = f" > {snakemake.output[0]}"
+if command in ("generate"):
+    output_args = f" --output {snakemake.output[0]}"
```

Ruff (0.15.18)

[error] Undefined name `snakemake`

(F821)